app/server/appsmith-server/src/main/java/com/appsmith/server/repositories/ce/CustomApplicationSnapshotRepositoryCEImpl.java

```diff
@@ -23,16 +23,15 @@ public CustomApplicationSnapshotRepositoryCEImpl(
 
     @Override
     public Mono<ApplicationSnapshot> findWithoutData(String applicationId) {
-        List<Criteria> criteriaList = new ArrayList<>();
-        criteriaList.add(
-                Criteria.where(ApplicationSnapshot.Fields.applicationId).is(applicationId));
-        criteriaList.add(Criteria.where(ApplicationSnapshot.Fields.chunkOrder).is(1));
+        BridgeQuery<ApplicationSnapshot> query = Bridge.query();
+        query.equal(ApplicationSnapshot.Fields.applicationId, applicationId)
+                .equal(ApplicationSnapshot.Fields.chunkOrder, 1);
```

Comment:
Minor, any reason we're explicitly creating the `.query()` instead of something like this? 🤔

```suggestion
        BridgeQuery<ApplicationSnapshot> query = Bridge.<ApplicationSnapshot>equal(ApplicationSnapshot.Fields.applicationId, applicationId)
                .equal(ApplicationSnapshot.Fields.chunkOrder, 1);
```

Reply:
Nothing i can think of.